docs/CONTRIBUTING.rst

```diff
@@ -16,11 +16,6 @@ Types of Contributions
 Did you find a bug? or Do you intend to add a new feature or change an existing one?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* **Identify the appropriate repository** for the change you are suggesting:
```

Comment:
OK, I do like having them separate so that the matnwb folks have a feed of the stuff that pertains to them as well. If we want to continue to have this be a rule then we need to follow it ourselves.  #552 was about ROI spec, and #626 and #594 were about the schema language... and there are about a dozen that I created that should probably be in schema as well.

Reply:
I agree.

Reply:
@oruebel @bendichter Is this conversation concluded? Can we merge this?